app/src/main/java/com/am/fdamilola/bankingapp/activities/Splash_Screen_Activity.java

```diff
@@ -0,0 +1,53 @@
+package com.am.fdamilola.bankingapp.activities;
+
+import android.content.Intent;
+import android.graphics.Typeface;
+import android.os.Bundle;
+import android.os.Handler;
+import android.widget.TextView;
+
+import com.am.fdamilola.bankingapp.R;
+import com.am.fdamilola.bankingapp.base.classes.TinyGameAppBaseActivity;
+
+public class Splash_Screen_Activity  extends TinyGameAppBaseActivity {
+
+    // Splash screen timer'
+    private TextView mSplashText ;
+    private static int SPLASH_TIME_OUT = 3000;
```

Comment:
Views should be inflated using ButterKnife, not findViewById.

Have a look at how to use butterknife. It's already part of the project.